app/withdrawal/mutations/withdraw.ts

```diff
@@ -11,8 +11,7 @@ import { Withdrawal } from '../validations'
 const withdraw = resolver.pipe(
   resolver.zod(Withdrawal),
   resolver.authorize(),
-  async ({ bank, account, amount }, ctx: Ctx) => {
-    if (!ctx.session.userId) throw new AuthorizationError()
+  async ({ bank, account, amount }, ctx) => {
```

Comment:
same here

Reply:
I'm leaving this as it is as it will conflict with Anoi's PR